cmd/ddev_gen_autocomplete/ddev_gen_autocomplete.go

```diff
@@ -14,7 +14,7 @@ func main() {
 		err = os.MkdirAll(targetDir, 0755)
 		util.CheckErr(err)
 	}
-	err := cmd.RootCmd.GenBashCompletionFile(filepath.Join(targetDir, "ddev_bash_completion.sh"))
+	err := cmd.RootCmd.GenBashCompletionFileV2(filepath.Join(targetDir, "ddev_bash_completion.sh"), false)
```

Comment:
There is also an option to generate descriptions:

```suggestion
	err := cmd.RootCmd.GenBashCompletionFileV2(filepath.Join(targetDir, "ddev_bash_completion.sh"), true)
```

The output will look like this:

```
$ ddev <TAB><TAB>
auth          (A collection of authentication commands)
clean         (Removes items DDEV has created)
completion    (Generate the autocompletion script for the specified shell)
composer      (Executes a Composer command within the web container)
...
```

@rfay let me know if we should use this feature.

Reply:
Sounds fantastic to me!